packages/plugin-owl/src/ui/OwlDomRenderer.ts

```diff
@@ -1,18 +1,28 @@
 import { DomRenderingEngine } from '../../../plugin-dom/src/DomRenderingEngine';
 import { AbstractRenderer } from '../../../plugin-renderer/src/AbstractRenderer';
+import { Renderer } from '../../../plugin-renderer/src/RenderingEngine';
 import { OwlNode } from './OwlNode';
 import { Owl } from '../Owl';
+import { OwlUIEnv } from './OwlUIComponent';
 
 export class OwlDomRenderer extends AbstractRenderer<Node[]> {
     static id = 'dom';
     engine: DomRenderingEngine;
     predicate = OwlNode;
+    env: OwlUIEnv;
+
+    constructor(engine: DomRenderingEngine, superRenderer: Renderer<Node[]>) {
+        super(engine, superRenderer);
+        this.engine = engine;
+        this.super = superRenderer;
+        const owlPlugin = this.engine.editor.plugins.get(Owl);
+        this.env = owlPlugin.owlEnv;
```

Comment:
At this point why not do `this.engine.editor.plugins.get(Owl).env` ? (So we would also rename `owlEnv`  to `env` since it is already in the plugin `Owl` anyway.)